README.md

```diff
@@ -178,4 +182,30 @@ Github Actions will run builds on branches after every push command. It will run
         curl -dH -X POST "$(terraform output webhook_url)"
 - Access Azure web-app at: http://module13-azure-terraform-sj.azurewebsites.net/
 - Update Loggly provisioning using Terraform with main.tf, variables.tf by adding "LOG_LEVEL" and "LOGGLY_TOKEN"
+```
```

Comment:
You've included minikube-darwin-amd64 in your github repo, but you should instead tell the reader to install minkube if they want it.

It's OS specific and a binary file that you don't maintain. No reason to include it here.